exosims.yml

```diff
@@ -0,0 +1,279 @@
+  - prompt_toolkit=3.0.36=hd3eb1b0_0
+  - ptyprocess=0.7.0=pyhd3eb1b0_2
+  - pure_eval=0.2.2=pyhd3eb1b0_0
+  - pycparser=2.21=pyhd3eb1b0_0
+  - pydot=1.4.2=py311h06a4308_0
+  - pylint=2.16.2=py311h06a4308_0
+  - pyopenssl=23.0.0=py311h06a4308_0
+  - pyparsing=2.4.7=pyhd8ed1ab_1
+  - pyqt=5.15.7=py311h6a678d5_0
+  - pyqt5-sip=12.11.0=py311h6a678d5_0
+  - pyrsistent=0.18.0=py311h5eee18b_0
+  - pysocks=1.7.1=py311h06a4308_0
+  - pytest=7.4.0=py311h06a4308_0
+  - python=3.11.2=h7a1cb2a_0
+  - python-dateutil=2.8.2=pyhd3eb1b0_0
+  - python-fastjsonschema=2.16.2=py311h06a4308_0
+  - python-tzdata=2023.3=pyhd3eb1b0_0
+  - python_abi=3.11=2_cp311
+  - qt-main=5.15.2=h8373d8f_8
+  - qt-webengine=5.15.9=hbbf29b9_6
+  - qtconsole=5.4.2=py311h06a4308_0
+  - qtpy=2.2.0=py311h06a4308_0
+  - qtwebkit=5.212=h3fafdc1_5
+  - readline=8.2=h5eee18b_0
+  - send2trash=1.8.0=pyhd3eb1b0_1
+  - setuptools=65.6.3=py311h06a4308_0
+  - sip=6.6.2=py311h6a678d5_0
+  - six=1.16.0=pyhd3eb1b0_1
+  - sniffio=1.2.0=py311h06a4308_1
+  - soupsieve=2.4=py311h06a4308_0
+  - sqlite=3.41.1=h5eee18b_0
+  - stack_data=0.2.0=pyhd3eb1b0_0
+  - terminado=0.17.1=py311h06a4308_0
+  - tinycss2=1.2.1=py311h06a4308_0
+  - tk=8.6.12=h1ccaba5_0
+  - toml=0.10.2=pyhd3eb1b0_0
+  - tomli=2.0.1=py311h06a4308_0
+  - tomlkit=0.11.1=py311h06a4308_0
+  - tornado=6.2=py311h5eee18b_0
+  - typing-extensions=4.5.0=py311h06a4308_0
+  - typing_extensions=4.5.0=py311h06a4308_0
+  - tzdata=2023c=h04d1e81_0
+  - urllib3=1.26.15=py311h06a4308_0
+  - websocket-client=0.58.0=py311h06a4308_4
+  - wheel=0.38.4=py311h06a4308_0
+  - widgetsnbextension=4.0.5=py311h06a4308_0
+  - wrapt=1.14.1=py311h5eee18b_0
+  - xarray=2023.6.0=py311h06a4308_0
+  - xarray-einstats=0.6.0=py311h06a4308_0
+  - xz=5.2.10=h5eee18b_1
+  - zeromq=4.3.4=h2531618_0
+  - zlib=1.2.13=h5eee18b_0
+  - zstd=1.5.4=hc292b87_0
+  - pip:
+    - absl-py==1.4.0
+    - astropy==5.2.2
+    - astroquery==0.4.6
+    - asttokens==2.2.1
+    - beautifulsoup4==4.12.1
+    - charset-normalizer==3.1.0
+    - comm==0.1.3
+    - cryptography==40.0.1
+    - debugpy==1.6.7
+    - executing==1.2.0
+    - exo-det-box==0.0.2
+    - h5py==3.8.0
+    - html5lib==1.1
+    - importlib-metadata==6.1.0
+    - ipykernel==6.22.0
+    - ipyparallel==8.5.1
+    - jaraco-classes==3.2.3
+    - jedi==0.18.2
+    - jeepney==0.8.0
+    - jplephem==2.18
+    - keyring==23.13.1
+    - meanstars==3.4.0
+    - more-itertools==9.1.0
+    - numpy==1.24.2
+    - ortools==9.6.2534
+    - pandas==2.0.0
+    - prompt-toolkit==3.0.38
+    - protobuf==4.22.1
+    - psutil==5.9.4
+    - pyerfa==2.0.0.3
+    - pygments==2.14.0
+    - pytz==2023.3
+    - pyvo==1.4.1
+    - pyyaml==6.0
+    - pyzmq==25.0.2
+    - requests==2.28.2
+    - scipy==1.10.1
+    - secretstorage==3.3.3
+    - stack-data==0.6.2
+    - synphot==1.2.0
+    - tqdm==4.65.0
+    - traitlets==5.9.0
+    - wcwidth==0.2.6
+    - webencodings==0.5.1
+    - zipp==3.15.0
+prefix: /home/pinchen/anaconda3/envs/exosims
```

Comment:
you will want to remove this before pushing to the repository or else others who generate the environment will have unexpected behavior on their machines

Reply:
Thank you for the tip!

Reply:
Agree with having a separate notebook folder, and removing unnecessary ipynb files.

Agree with having one "ebs" environment.  We should remove the exosims*.yml files (they were for debugging).  FYI, conda failed to install <emcee> on a machine in the ebs env (not sure what caused the conflict).  It would be helpful to list minimum required packages to install for users who want to create/use their own conda environment.

Reply:
sounds good! I am happy to work on that and make a cleaner environment file with only the necessary packages and also keep track of any packages that need to be installed by the user beforehand so that we can put those in the README